Models/Project.cs

```diff
@@ -61,6 +62,7 @@ public Project()
 
         public CallToAction CallToAction { get; set; }
 
+        [JsonIgnore]
```

Comment:
Why is this here and what purpose does it serve?

Reply:
Newtonsoft.Jsonconvert created an infinite loop when we tried to send the project to the messagebroker.